.github/workflows/run-integration-tests-test-resources.yml

```diff
@@ -39,11 +43,22 @@ jobs:
         env:
           NODE_AUTH_TOKEN: ${{ secrets.npm-token }}
 
+      - name: Install dependencies
```

Comment:
This is fine, just wondering if we could do some caching

Reply:
I think the `cache: npm` is enough or is there something else?